hoot-core/src/main/cpp/hoot/core/io/OsmPbfWriter.cpp

```diff
@@ -238,7 +238,8 @@ void OsmPbfWriter::_open(const QString& url)
 void OsmPbfWriter::open(const QString& url)
 {
   _open(url);
-  initializePartial();
+  //  OsmPbfWriter::open() shouldn't call OsmPbfWriter::initializePartial()
+  //  or it will cause duplicate headers
```

Comment:
This is the big issue, `open()` called `initializePartial()` that writes the header and then when a file is streamed it is called twice.  Remove it here.